src/Commands/ToggleShifter.cpp

```diff
@@ -0,0 +1,15 @@
+#include "ToggleShifter.h"
+#include "Subsystems/Chassis.h"
+
+ToggleShifter::ToggleShifter() {
+  Requires(Chassis::getInstance().get());
+	// Use Requires() here to declare subsystem dependencies
+	// eg. Requires(Robot::chassis.get());
+}
+
+// Called once when the command executes
+void ToggleShifter::Initialize() {
+  Chassis::ShifterValue value
+    = Chassis::getInstance()->IsShifterHigh() ? Chassis::kShifterHigh : Chassis::kShifterLow;
```

Comment:
This logic seems reversed. If the shifter is high, make the shifter high? This will never shift.